workflow-templates/solutiontemplate.pr.yml

```diff
@@ -28,23 +29,24 @@ jobs:
           armttkVersion: aka.ms/arm-ttk-latest
   RunTests:
     runs-on: windows-latest
+    needs: ValidateARMTemplates
```

Comment:
Whoops - this was removed when Dan's PR was merged. Let's remove this requirement.

Reply:
As discussed offline, we will remove all dependencies between the jobs in the application workflow templates.